examples/tools/dialog/run

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Why is this needed? Can't we just have the Python script with executable flags? The `/usr/bin/env python` bit should take care of the interpreter, and there's nothing in there that needs `PYTHONPATH` to be set to anything special?

Reply:
This was a copy of my setup/. On windows its not so easy. On Linux for sure. Lets do it the easy way.